src/index.js

```diff
@@ -36,10 +36,14 @@ class KadDHT {
    * @param {number} options.kBucketSize k-bucket size (default 20)
    * @param {Datastore} options.datastore datastore (default MemoryDatastore)
    * @param {boolean} options.enabledDiscovery enable dht discovery (default true)
+   * @param {object} options.validators validators object with namespace as keys and function(key, record, callback)
+   * @param {object} options.selectors selectors object with namespace as keys and function(key, records)
    */
   constructor (sw, options) {
     assert(sw, 'libp2p-kad-dht requires a instance of Switch')
     options = options || {}
+    options.validators = options.validators || {}
+    options.selectors = options.selectors || {}
```

Comment:
This is unnecessary isn't it? The spread operator will ignore `undefined` and `null`.

Reply:
Just tested it. You are right, removed the fallback 😀